WhatsApp/app/src/main/res/layout/item_chat.xml

```diff
@@ -6,7 +6,30 @@
     android:layout_height="wrap_content"
     android:background="?selectableItemBackground"
     android:clickable="true"
-    android:focusable="true">
+    android:focusable="true"
+    android:paddingTop="10dp"
```

Comment:
we can use icon_padding_10 in dimens.xml. even better to rename it to dp10 for generic use